site/content/docs/howto/concierge/configure-concierge-supervisor-jwt.md

```diff
@@ -5,7 +5,7 @@ cascade:
   layout: docs
 menu:
   docs:
-    name: JWT Authentication with Supervisor
+    name: With Supervisor
```

Comment:
Word smash, so I adjusted given our Supervisor section is heavily utilizing the `with xyz` pattern now.
Index page title is still full text.

<img width="589" alt="Screenshot 2023-09-05 at 4 26 17 PM" src="https://github.com/vmware-tanzu/pinniped/assets/280512/dfbfe0e1-2125-4d2f-950d-bcb05b58c326">